resources/compile-config.sh

```diff
@@ -5,19 +6,18 @@ cache_dir="$HOME/Library/Caches/com.runningwithcrayons.Alfred-2/Workflow Data/co
 mkdir -p "$cache_dir"
 
 # Get paths to configuration file
-installed_config="./config.applescript"
-cached_config="$cache_dir/config.applescript"
+installed_config="config.applescript"
 compiled_config="$cache_dir/config.scpt"
+cached_config_md5file="$cache_dir/config.applescript.md5"
 
-# Retrieve contents of config files
-installed_config_contents="$(cat "$installed_config" 2> /dev/null)"
-cached_config_contents="$(cat "$cached_config" 2> /dev/null)"
+# Retrieve md5sum of config files
+installed_config_md5sum=$(md5 "$installed_config" 2> /dev/null)
+cached_config_md5sum=$(< "$cached_config_md5file")
 
-# If cached config does not match installed config
-if [ "$installed_config_contents" != "$cached_config_contents" ]; then
-
-	# Cache installed config
-	cp "$installed_config" "$cached_config"
-	echo "$installed_config_contents" | osacompile -o "$compiled_config"
+# If cached md5sum does not match installed config's md5sum
+if [ "$installed_config_md5sum" != "$cached_config_md5sum" ]; then
```

Comment:
Could you please add a newline below this line? I prefer to have newlines surrounding the start/end lines for blocks (at least for `play-song`).